packages/phrases/package.json

```diff
@@ -25,7 +25,6 @@
     "url": "https://github.com/logto-io/logto/issues"
   },
   "dependencies": {
-    "@logto/schemas": "^0.1.0",
```

Comment:
Why do we have cyclic warning in the first place? Does `@logto/schemas` also depend on `phrases`?

Reply:
yes `schemas` depend on `phrases`, while `phrases` actually doesn't need `schemas`

Reply:
Yeah, makes sense